app/src/main/java/org/oppia/android/app/profile/AdminSettingsDialogFragmentPresenter.kt

```diff
@@ -40,18 +41,17 @@ class AdminSettingsDialogFragmentPresenter @Inject constructor(
       viewModel = adminViewModel
     }
 
-    binding.inputPin.setInput(adminViewModel.inputPin.get().toString())
-    binding.inputPin.addTextChangedListener(object : TextWatcher {
-      override fun onTextChanged(confirmPin: CharSequence?, start: Int, before: Int, count: Int) {
-        confirmPin?.let {
-          adminViewModel.inputPin.set(confirmPin.toString())
-          adminViewModel.errorMessage.set("")
-        }
+    val previousErrorMessage = adminViewModel.errorMessage.get()
```

Comment:
need to check if this is working or not, as the similar logic is not working in other pr.

Reply:
No, it's not working

Reply:
I've created a new implementation for this 
https://github.com/oppia/oppia-android/pull/2349#discussion_r553853573 check the explanation here